fontgenerator.js

```diff
@@ -435,6 +433,11 @@ TinyFontGenerator = {
     data += " 00 00 00 00";  // italicAngle
     data += " 00 00 00 00";  // underlinePosition
     data += " 00 00 00 00";  // underlineThickness
+    data += " 00 00 00 00";  // isFixedPitch
```

Comment:
Leaving these `post` values out should not lead to an OTS error, though. Every browser that I tried works just fine without them, which is why they're left out (since this is a generator for a *minimal* working font)